tests/CMakeLists.txt

```diff
@@ -5,7 +5,7 @@ include_directories( "${CMAKE_SOURCE_DIR}/plugins/wallet_plugin/include" )
 file(GLOB UNIT_TESTS "*.cpp")
 
 add_executable( plugin_test ${UNIT_TESTS} )
-target_link_libraries( plugin_test eosio_testing eosio_chain chainbase chain_plugin wallet_plugin fc ${PLATFORM_SPECIFIC_LIBS} )
+target_link_libraries( plugin_test eosio_testing eosio_chain chainbase chain_plugin wallet_plugin fc net_plugin ${PLATFORM_SPECIFIC_LIBS} )
```

Comment:
Why did you need to add this?

Reply:
When I initially developed the unit test, I placed it in the tests directory.  When we decided to create a test directory under net_plugin for unit tests, I did not backtrack and remove the change.  The file has been modified back to the original.